runtime/src/snapshot_utils/archive_format.rs

```diff
@@ -21,11 +23,25 @@ impl ArchiveFormat {
             ArchiveFormat::TarBzip2 => TAR_BZIP2_EXTENSION,
             ArchiveFormat::TarGzip => TAR_GZIP_EXTENSION,
             ArchiveFormat::TarZstd => TAR_ZSTD_EXTENSION,
+            ArchiveFormat::TarLz4 => TAR_LZ4_EXTENSION,
             ArchiveFormat::Tar => TAR_EXTENSION,
         }
     }
 }
 
+/// Converts to a String
+impl ToString for ArchiveFormat {
```

Comment:
yeah. kind of unfortunate that we can't be symmetric here because their usage is slightly different. 
FromStr() is used to autodetect the compression method used to decompress the archives when downloaded remotely.  ToStr() is used to report metrics. Therefore, the strings used in these two functions are slightly different.